app/ai/voice/agents/breeze_buddy/managers/webhook.py

```
@@ -0,0 +1,78 @@
+            success = await send_webhook_with_retry(
+                session, reporting_webhook_url, summary_data, max_retries=3
+            )
+            if success:
+                logger.info(
+                    f"Successfully sent call summary webhook on no_answer (attempt {lead.attempt_count + 1}, isLastAttempt: {is_last_attempt})."
+                )
+            else:
+                logger.error(
+                    "Failed to send call summary webhook on no_answer after all retries."
+                )
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)